ts/sagas/startup/checkAcknowledgedEmailSaga.ts

```diff
@@ -0,0 +1,39 @@
+import { Effect } from "redux-saga";
+import { put, take } from "redux-saga/effects";
+
+import { UserProfileUnion } from "../../api/backend";
+import { navigateToEmailValidationScreen } from "../../store/actions/navigation";
+import { emailAcknowledged } from "../../store/actions/onboarding";
+
+/**
+ * Launch email saga that consists of:
+ * - acknowledgement screen if email already exists and is valid and the user
+ *   does not want to change it
+ * - editing and validation screen, otherwise
+ */
+export function* checkAcknowledgedEmailSaga(
+  userProfile: UserProfileUnion
+): IterableIterator<Effect> {
+  // Check if user profile has email
+  if (userProfile.spid_email && userProfile.spid_email !== "") {
```

Comment:
Some of here https://github.com/teamdigitale/io-app/pull/1308#discussion_r327025006

Reply:
question asked here #1308

Reply:
questions answered here #1308